sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/AdvancedDiffLabeledUnlabeledDataAsync.java

```diff
@@ -112,23 +137,37 @@ public static void main(String[] args) throws IOException {
             unLabeledForm.getFields().forEach((label, formField) -> {
                 final StringBuilder boundingBoxStr = new StringBuilder();
                 if (formField.getValueText().getBoundingBox() != null) {
-                    formField.getValueText().getBoundingBox().getPoints().forEach(point ->
-                        boundingBoxStr.append(String.format("[%.2f, %.2f]", point.getX(), point.getY())));
+                    formField.getValueText().getBoundingBox().getPoints().stream().map(point ->
+                        String.format("[%.2f, %.2f]", point.getX(), point.getY())).forEach(boundingBoxStr::append);
                 }
 
                 final StringBuilder boundingBoxLabelStr = new StringBuilder();
                 if (formField.getLabelText() != null && formField.getLabelText().getBoundingBox() != null) {
-                    formField.getLabelText().getBoundingBox().getPoints().forEach(point ->
-                        boundingBoxLabelStr.append(String.format("[%.2f, %.2f]", point.getX(), point.getY())));
+                    formField.getLabelText().getBoundingBox().getPoints().stream().map(point ->
+                        String.format("[%.2f, %.2f]", point.getX(), point.getY())).forEach(boundingBoxStr::append);
                 }
                 System.out.printf("Field %s has label %s  within bounding box %s with a confidence score "
                         + "of %.2f.%n",
                     label, formField.getLabelText().getText(), boundingBoxLabelStr, formField.getConfidence());
 
-                System.out.printf("Field %s has value %s based on %s within bounding box %s with a confidence score "
-                        + "of %.2f.%n",
+                System.out.printf("Field %s has value %s based on %s within bounding box %s with a confidence "
+                        + "score of %.2f.%n",
                     label, formField.getFieldValue(), formField.getValueText().getText(), boundingBoxStr,
                     formField.getConfidence());
+
+                // Find the value of a specific unlabeled field. The specific key "Vendor Name:" provided in the
+                // example will only be found if sample training forms used
+                unLabeledForm.getFields().entrySet()
+                    .stream()
+                    //filter by label text
+                    .filter(formFieldEntry -> "Vendor Name:".equals(formFieldEntry.getValue().getLabelText().getText()))
+                    .findAny()
+                    .ifPresent(formFieldEntry ->
+                        System.out.printf("The Vendor name is: %s%n", formFieldEntry.getValue()));
+                    // @since 9
+                    // .ifPresentOrElse(
+                    //     formFieldEntry -> System.out.printf("The Vendor name is: %s%n", formFieldEntry.getValue()),
```

Comment:
remove?

Reply:
I think with Java samples we want to start showing the easier way of doing things which are usually with higher versions of the Java language. But since our support needs to be catered to atleast Java 8 we show the Java 8 way and other way is commented.

Reply:
ohhh and does ` @since 9` signals that? or should there be a comment with that info?

Reply:
yes,  that's a common Javadoc that signals the Java version info.